stories/panel.stories.ts

```diff
@@ -129,4 +145,45 @@ storiesOf('Basics/Panel', module)
         }),
       ]);
     })
+  )
+  .add(
+    'collapsed unless canRefine',
+    withHits(
+      ({ search, container }) => {
+        const breadcrumbInPanel = panel({
+          collapsed: ({ widgetRenderState }) => {
+            // TODO: We need better typing here. Not sure how, yet.
+            return widgetRenderState && !(widgetRenderState as any).canRefine;
```

Comment:
```suggestion
            return widgetRenderState && !(widgetRenderState as any).canRefine;
```

in which case is widgetRenderState undefined or null?

Reply:
when it's a custom widget provided by user https://github.com/algolia/instantsearch.js/blob/feat/widget-render-state-in-panel/src/types/widget.ts#L324:L324

Reply:
good call, we can't force them to return it, but I wonder if we could default it to an empty object to prevent this condition? not sure if it's a good idea though

Reply:
https://github.com/algolia/instantsearch.js/pull/4527/commits/ab6fd0a64a9ded34758677f018a38b756ed1089c

Yeah I think it's good to provide an empty object by default, so that users don't have to deal with nullish check.